CHANGELOG/CHANGELOG-v0.4.md

```diff
@@ -0,0 +1,23 @@
+# Changelog since v0.3.0
+All notable changes to this project will be documented in this file.
+
+## v0.4.0
+
+### New Features 
+* Support managedClusterAddon registration and maintain the lease of managedClusterAddon.
+
+### Added
+* Upgrade CRD to v1 and k8s lib to v0.21.0-rc.0.
+* Upgrade Go to 1.16.
+* Support DNS names for client cert.
+
+### Changes
+* Merge managed cluster capacity status.
+* Reset csr name only if csr sync failed or secret saved.
+
+### Bug Fixes
+* Fix the managedClusterAddon lease compatibility issue.
+* Fix the invalid client certificate issue.
```

Comment:
We may remove `Fix the invalid client certificate issue.`, this should belong to v0.3.0